code/__DEFINES/is_helpers.dm

```diff
@@ -4,7 +4,14 @@
 
 #define islist(L) (istype(L, /list))
 
+#if DM_VERSION >= 512
+#define in_range(source, user) (get_dist(source, user) <= 1 && (get_step(source, 0)?:z) == (get_step(user, 0)?:z))
```

Comment:
these ```get_step()```s should be ```get_turf()```, for consistency reasons.
```get_turf(X)``` is just ```get_step(X, 0)``` anyway.

Reply:
This file comes before `get_turf` is defined.

Reply:
For the unenlightened, what does ?: do?

Reply:
>[10:21:09] 18<25MrStonedOne18> blah?:var if (blah) blah:var

Reply:
too late now, but defines within defines are resolved when the define is used, not when the define is defined.

so this can use get_turf no issue.